backend/s3/s3.go

```diff
@@ -2644,6 +2654,17 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 	if opt.BucketACL == "" {
 		opt.BucketACL = opt.ACL
 	}
+	if opt.SSECustomerKeyB64 != "" && opt.SSECustomerKey != "" {
+		return nil, errors.New("s3: can't use sse_customer_key and sse_customer_key_base64 at the same time")
+	}
+	if opt.SSECustomerKeyB64 != "" && opt.SSECustomerKey == "" {
```

Comment:
The `&& opt.SSECustomerKey == ""` is redundant due to the if above I think.

Reply:
Changed -- I also made it an `else if`, which is technically irrelevant, but I feel like it makes the code more clear that those two are related and the following one is not (or not directly, anyway -- it's about making sure we have the md5, not about the key field itsself)